### DISCUSSION TITLE
Postgres database not visible

### DISCUSSION THREAD

**Discussion by brunoalreis** [dbeaver/dbeaver#16103] [Q&A]
#### System information: 
- Windows 11
- DBeaver version 22.0.2.202204021640

#### Connection specification:
- Postgresql 14
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:

Not possible to see the following database: Test.Api.Local
When I create a new database with this name, it is visible. If I Refresh databases, it disappears.

#### Steps to reproduce, if exist:
1 - Create database with name: Test.Api.Local
2 - Refresh databases

**Comment by LonwoLonwo** [comment]
Hello @brunoalreis 

By default, DBeaver shows only one database.

So maybe enabling the "Show all databases" in Connection Settings can help you?

![2022-04-05 17_21_08-Window](https://user-images.githubusercontent.com/45152336/161775950-c9e5a186-91a8-47bb-8864-7a32208f5939.png)

**Comment by brunoalreis** [comment]
Yes, it worked.

Thanks.

**Comment by LonwoLonwo** [comment]
Thanks for the answer @brunoalreis
